client/js/libs/jquery/stickyscroll.js

```diff
@@ -16,7 +24,8 @@
 					self.scrollBottom();
 				}
 			})
-			.scrollBottom();
+			.scrollBottom()
+			.data("window-sticky-fn", windowResizeHandler);
```

Comment:
We do unbind it, but I think it's better to not assume so to avoid weird surprises. I don't usually expect a jQuery plugin bound on an element to have an effect on other elements... Someone unaware of that quirk would get really confused at our little library :)